dashboard-frontend/src/hooks/useStudyConf.tsx

```diff
@@ -0,0 +1,26 @@
+import { useQuery, queryCache } from 'react-query';
+import useAuthenticatedApi from './useAuthenticatedApi';
+
+const useStudyConf = (slug: string) => {
+  const useStudyConfQuery = (slug: string) => {
```

Comment:
Not sure I understand why this is defined as a function

Reply:
Fair point, just following previous code patterns, have refactored to do away with the function and set `studyConfQuery` as a variable with a value of `useQuery(['studyConf', slug], () =>
    fetchStudyConf({ slug }));`